.github/workflows/docker-release.yml

```
@@ -0,0 +1,47 @@
+name: Docker release
+
+on:
+  push:
+    tags:
+      - v*
+  pull_request:
+
+jobs:
+  # Build the project as a Docker image and push it to Docker Hub
+  push_to_registry:
+    name: Push Docker image to Docker Hub
+    runs-on: ubuntu-latest
+    outputs:
+      tag: ${{ steps.meta.outputs.tags }}
+      label: ${{ steps.meta.outputs.labels }}
+    steps:
+      - name: Check out the repo
+        uses: actions/checkout@v3
+      - name: Check Docker Version
+        run: docker --version
+      - name: Log in to Docker Hub
+        uses: docker/login-action@v2
+        with:
+          username: ${{ vars.DOCKER_USERNAME }}
+          password: ${{ secrets.DOCKER_PASSWORD }}
+      - name: Extract metadata (tags, labels) for Docker
+        id: meta
+        uses: docker/metadata-action@v4
+        with:
+          images: ${{ vars.DOCKER_USERNAME }}/saucectl
```

We have an org we can publish under:
```suggestion
          images: saucelabs/saucectl
```

---

@alexplischke are you able to retrieve the password for `saucelabs` and set it in our secrets?

---

Yup, I can!